Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -543,7 +537,7 @@ def bottle_formula(f, args:)
           "date"     => Pathname(local_filename).mtime.strftime("%F"),
           "tags"     => {
             bottle_tag.to_s => {
-              "filename"              => bottle_filename,
+              "filename"              => filename.url_encode,
```

Comment:
Could even consider a rename to 
```suggestion
              "remote_filename"              => filename.url_encode,
```

Reply:
I agree but backwards compatibility seems to be a concern here.

Reply:
@Bo98 Could consider duplicating it using both names; may give us options in future?

Reply:
Maybe but there's not really a way to "deprecate" JSON fields.

Reply:
Yes but given this JSON is unversioned and undocumented I think we could consider just changing it ourselves.